one-ui/src/main/java/com/marklogic/hub/oneui/auth/LoginLogoutHandler.java

```diff
@@ -43,19 +42,35 @@ public void onAuthenticationSuccess(HttpServletRequest request, HttpServletRespo
         ConnectionAuthenticationToken authenticationToken = (ConnectionAuthenticationToken)authentication;
         ObjectNode resp = mapper.createObjectNode();
         boolean isInstalled = authenticationToken.stagingIsAccessible();
+        final boolean[] managePrivilege = new boolean[3];
         if (authenticationToken.stagingIsAccessible()) {
             try {
                 RolesService rolesService = RolesService.on(authenticationToken.getHubConfigSession().newStagingClient());
                 resp.putArray("roles").addAll((ArrayNode) rolesService.getRoles());
+                if (authenticationToken.hasManagePrivileges()) {
+                    ArrayNode rolesNodes = (ArrayNode) resp.get("roles");
+                    rolesNodes.forEach(e -> {
+                        if ("data-hub-environment-manager".equals(e.asText())) {
+                            managePrivilege[0] = true;
+                            return;
+                        } else if ("manager-admin".equals(e.asText())) {
```

Comment:
This should be "manage-admin".
manage-admin, security and data-hub-environment-manager are roles and so managePrivilege should probably be called hasManageAdminAndSecurityRole? 
final variable should be all capital.
`rolesNodes.forEach(role ->` is more readable